sensor/common/compliance/service_impl.go

```diff
@@ -143,7 +145,9 @@ func (s *serviceImpl) Communicate(server sensor.ComplianceService_CommunicateSer
 			s.auditEvents <- t.AuditEvents
 			s.auditLogCollectionManager.AuditMessagesChan() <- msg
 		case *sensor.MsgFromCompliance_NodeScanV2:
-			log.Infof("NodeScanV2 message received: %v", msg)
```

Comment:
Up to you, but want to keep this as a Debug log if that'll help development?

Reply:
We will work more on the sensor corner, so there will be a chance to add some more meaningful debugging